### ISSUE TITLE
Control.TextInput ref returns null

### ISSUE THREAD

**Issue by bgriffith** [davidkpiano/react-redux-form#555]
Small issue I've recently come up against when trying to trigger an input to focus is that when I use `Control.TextInput` for a field in React Native, `ref` will only return `null`. Is there an alternative way of doing it or is this a bug?

```js
<Control.TextInput ref={input => console.log(input)} />
```

**Comment by davidkpiano** [comment]
Hey @bgriffith, that might be a bug - I'll investigate and fix it. In any case, the implementation is small for binding to React Native components:

```jsx
Control.TextInput = (props) => (
  <Control
    component={TextInput}
    mapProps={{
      onResponderGrant: ({ onFocus }) => onFocus,
      value: (_props) => ((!_props.defaultValue && !_props.hasOwnProperty('value'))
        ? getTextValue(_props.viewValue)
        : _props.value),
      onChangeText: ({ onChange }) => onChange,
      onChange: noop,
      onBlur: ({ onBlur, viewValue }) => () => onBlur(viewValue),
      onFocus: ({ onFocus }) => onFocus,
    }}
    {...props}
  />
);
```

so if you'd like to figure out what's missing, perhaps you can make a quick PR and I can add it in. Also, I believe that for now, this will work:

```jsx
<Control.TextInput controlProps={{ref: input => console.log(input)}} />
```

but ideally it should work the way you have it.

**Comment by bgriffith** [comment]
Hey @davidkpiano Thanks for the tips. Weird things be happening though! If I add the `controlProps` prop to the component (doesn't matter what I include within it) the whole input disappears from view!

**Comment by davidkpiano** [comment]
Since React doesn't support propagating the `ref` prop through wrapped components (for whatever reason...), I think the solution is for me to add `getRef={(node) => ...}` to the API.

Does the name `getRef` sound intuitive enough? Or should I name it something different, like `controlRef`? It just can't be named `ref`.

**Comment by bgriffith** [comment]
`getRef` sounds intuitive enough to me :+1:

**Comment by davidkpiano** [comment]
Okay, done! Published as 1.3.2.

**Comment by bgriffith** [comment]
Awesome thanks a lot! When you a min could you publish to npm?

**Comment by davidkpiano** [comment]
Done! Published as 1.3.2
